app/src/main/java/io/homeassistant/companion/android/launch/LaunchActivity.kt

```diff
@@ -41,4 +43,8 @@ class LaunchActivity : AppCompatActivity(), LaunchView {
         presenter.onFinish()
         super.onDestroy()
     }
+
+    override fun attachBaseContext(newBase: Context?) {
```

Comment:
We should wrap all activity.
Or create a Base Activity which I do not like.

>We will need to inject the Lokalise SDK into the Activity context as well. To do so, we recommend you create a base Activity class and extend all your activities from it. Add the following code into your activity:

Reply:
Will add to every activity